Change the name of the executable file from Squeak to Pharo
--- a/platforms/unix/vm-display-X11/sqUnixX11.c
+++ b/platforms/unix/vm-display-X11/sqUnixX11.c
@@ -153,8 +153,8 @@
 /*** Variables -- X11 Related ***/
 
 /* name of Squeak windows in Xrm and the WM */
-#define xResClass	"Squeak"
-#define xResName	"squeak"
+#define xResClass	"pharo-vm"
+#define xResName	"Pharo"
 
 char		*displayName= 0;	/* name of display, or 0 for $DISPLAY */
 Display		*stDisplay= null;	/* Squeak display */
